server/lib/game/question_cache.ex

```diff
@@ -33,15 +33,18 @@ defmodule Game.QuestionCache do
   end
 
   def get_questions(questions, pack) do
-    case pack == "Variety" do
-      true -> 
-        questions |> get_random_questions(10)
-      false -> 
-        Enum.filter(questions, fn x ->
-          pack_field = Enum.at(x, 2)
-          pack_name = Enum.at(pack_field, 0)
-          pack_name == pack
-        end) |> get_random_questions(10)
+    case Enum.empty?(questions) do 
+      true -> Database.Catalog.random_formatted_questions(10, pack)
+      false ->
+        case pack == "Variety" do
+          true -> questions |> get_random_questions(10)
+          false -> 
+            Enum.filter(questions, fn x ->
+                  pack_field = Enum.at(x, 2)
```

Comment:
we should probably set up some sort of linting

shouldn't this block be indented back? from lines 43 and 46